task1/css_form.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Регистрация </title>
+    <link href="css_form.css" rel="stylesheet">
+</head>
+
+<body>
+    <form name="form" method="post" action="register.php" class="container">
```

Comment:
It is not a good idea to have a class and id with the same name. You could easily confuse them.